Traceback (most recent call last):
  File "C:\Users\TESTUSER\AppData\Local\Programs\Python\Python310\lib\site-packages\jupyter_cache\executors\utils.py", line 58, in single_nb_execution
    executenb(
  File "C:\Users\TESTUSER\AppData\Local\Programs\Python\Python310\lib\site-packages\nbclient\client.py", line 1263, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "C:\Users\TESTUSER\AppData\Local\Programs\Python\Python310\lib\site-packages\nbclient\util.py", line 85, in wrapped
    return just_run(coro(*args, **kwargs))
  File "C:\Users\TESTUSER\AppData\Local\Programs\Python\Python310\lib\site-packages\nbclient\util.py", line 60, in just_run
    return loop.run_until_complete(coro)
  File "C:\Users\TESTUSER\AppData\Local\Programs\Python\Python310\lib\asyncio\base_events.py", line 641, in run_until_complete
    return future.result()
  File "C:\Users\TESTUSER\AppData\Local\Programs\Python\Python310\lib\site-packages\nbclient\client.py", line 701, in async_execute
    await self.async_execute_cell(
  File "C:\Users\TESTUSER\AppData\Local\Programs\Python\Python310\lib\site-packages\nbclient\client.py", line 1019, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "C:\Users\TESTUSER\AppData\Local\Programs\Python\Python310\lib\site-packages\nbclient\client.py", line 913, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
# INCORRECT
DavidCox = date.fromisoformat("1924-7-15")
------------------

[1;31m---------------------------------------------------------------------------[0m
[1;31mValueError[0m                                Traceback (most recent call last)
Cell [1;32mIn [9], line 2[0m
[0;32m      1[0m [38;5;66;03m# INCORRECT[39;00m
[1;32m----> 2[0m DavidCox [38;5;241m=[39m [43mdate[49m[38;5;241;43m.[39;49m[43mfromisoformat[49m[43m([49m[38;5;124;43m"[39;49m[38;5;124;43m1924-7-15[39;49m[38;5;124;43m"[39;49m[43m)[49m

[1;31mValueError[0m: Invalid isoformat string: '1924-7-15'
ValueError: Invalid isoformat string: '1924-7-15'

